Battle Royal.playground/Contents.swift

```diff
@@ -0,0 +1,488 @@
+import UIKit
+
+protocol Resource{
+    var name: String { get }
+    
+    var damageBonus: Int { get }
+    var healBonus: Int {get}
```

Comment:
Нужно придерживаться одного стиля
```suggestion
    var healBonus: Int { get }
```